src/piadm.sh

```diff
@@ -765,10 +788,10 @@ bootable() {
 				if [[ $? -eq 0 && \
 					-f $tdir/EFI/Boot/bootx64.efi ]]; then
 					efi="and UEFI"
-					umount $tdir
 				else
 					efi=""
 				fi
+				umount -f $tdir > /dev/null 2>&1
```

Comment:
This could be:

    umount -f "$tdir" > /dev/null 2>&1 && rmdir "$tdir"

This way, rmdir is only attempted if the `umount` succeeded, which is probably safer.

Reply:
Thank you.